packages/material/src/complex/MaterialObjectRenderer.tsx

```diff
@@ -54,17 +54,12 @@ export const MaterialObjectRenderer = ({
         schema,
         uischema.scope,
         path,
-        'Group',
+        () =>  {return isEmpty(path)?Generate.uiSchema(schema, 'VerticalLayout'):{...Generate.uiSchema(schema, 'Group'), label: label}},
         uischema,
         rootSchema
       ),
     [uischemas, schema, uischema.scope, path, uischema, rootSchema]
```

Comment:
When the label changes we might need to generate a new UI Schema, therefore the label needs to be added to the dependency array
```suggestion
    [uischemas, schema, uischema.scope, path, label, uischema, rootSchema]
```